app/views/layouts/_utility_menu.erb

```diff
@@ -0,0 +1,95 @@
+<!-- Desktop Utility Menu -->
+<div class="bg-primary-darker w-full display-none desktop:display-flex flex-justify-end padding-1 font-body-2xs text-white">
+    <div class="display-flex flex-align-center margin-x-1">
+        <%= image_tag('images/usa-icons/grid_view.svg', class: "usa-icon--size-3 icon-white margin-top-1 margin-right-1", alt: "Dashboard") %>
+        <a href="/dashboard" class="text-white margin-top-1">
+        Dashboard
+        </a>
+    </div>
+    <% if current_user.role == "challenge_manager" %>
+        <div class="display-flex flex-align-center margin-x-1">
```

Comment:
can we make this full menu item area clickable (including the icon)? this might need to change from a div to a button or a tag so that it's tabbed using a keyboard.